krave/ui/ui.py

```diff
@@ -2,12 +2,10 @@
 import matplotlib.pyplot as plt
 #import numpy as np
 import pandas as pd
-from button_class import Button
-from button_refresh_class import RefreshButton
-from krave.ui.constants import Colors, PATHS, DEFAULT_FPS, DEFAULT_UPDATE_TIME_SECONDS, DATA_HEADERS
+#from krave.ui.constants import Colors, PATHS, DEFAULT_FPS, DEFAULT_UPDATE_TIME_SECONDS, DATA_HEADERS
```

Comment:
Add a TODO here to fix package import
#TODO(r.hueto@icloud.com): Figure out how to import krave.ui as a package.